### PR TITLE
Batch load initial changes

### PR DESCRIPTION
**Pull request by rroohhh**
This seems to improve the initial load performance for me.
Going from ~1.6 seconds to 1.1 seconds.

### CODE REVIEW

File: frontend/src/editor/webvtt_export.tsx [hunk 1]
```diff
@@ -24,7 +24,7 @@ export function WebvttExportModal({
   const [format, setFormat] = useState('vtt' as SubtitleFormat);
 
   return (
-    <Modal {...props} onClose={onClose} label="Export as WebVTT">
+    <Modal {...props} onClose={onClose} label="Export as">
```
Comment (pajowu) [inline frontend/src/editor/webvtt_export.tsx:27]: Maybe just `Export`?

Comment (rroohhh) [reply frontend/src/editor/webvtt_export.tsx:27]: I am not that tied to this title, but `Export as` seems a common title for dialogs that let you choose the export format. 
Furthermore it acts as a title for the format dropdown, which I like.

Comment (anuejn) [reply frontend/src/editor/webvtt_export.tsx:27]: then I would at least add ...

Comment (pajowu) [reply frontend/src/editor/webvtt_export.tsx:27]: `…` is even better

**Pull request merged**